frontend/src/components/managePermissions/table/TablePlaceholder.vue

```diff
@@ -1,8 +1,16 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import { IconSize } from '@/enum/IconEnum';
+</script>
 
 <template>
     <div class="no-app-selected">
-        <IconCapitol role="presentation" title="Icon Capitol" />
+        <Icon
+            role="presentation"
+            icon="search--locate"
+            :size="IconSize.small"
+            class="icon-color"
```

Comment:
Hi Nick, later we might add other style to this icon, let's call this class "placeholder-icon" instead of "icon-colour", it's too specific

Reply:
Sounds good. I'll replace it. Thanks!